cellengine/resources/experiment.py

```diff
@@ -209,28 +210,46 @@ def undelete(self):
             del self.deleted
 
     @property
-    def active_compensation(self) -> Optional[Union[Compensation, int]]:
+    def active_compensation(self) -> Optional[Union[str, int]]:
+        """The most recently used compensation.
+
+        Accepted (case-insensitive) values are one of:
+            - A Compensation object,
+            - A Compensation ID,
+            - A `cellengine` module variable, one of: [UNCOMPENSATED,
+                FILE_INTERNAL, PER_FILE]
+            - One of [0, -1, -2]
+
+        Example:
+            ```python
+            import cellengine
+            exp = cellengine.get_experiment(name='my experiment')
+            experiment.active_compensation = cellengine.UNCOMPENSATED
+            ```
+        """
```

Comment:
This doesn't render as a list because there's no space before the first list item:
![image](https://user-images.githubusercontent.com/469365/167749079-769096c8-53c5-475c-9ad4-7d5843f5d755.png)

and I'd like to clarify it.

```suggestion
        """The most recently used compensation.
        
        Accepted values are:

        - A Compensation object (value will be set to its `_id`)
        - A Compensation ID
        - A `cellengine` Compensation constant:
          [`UNCOMPENSATED`][cellengine.UNCOMPENSATED],
          [`FILE_INTERNAL`][cellengine.FILE_INTERNAL] or
          [`PER_FILE`][cellengine.PER_FILE].

        Example:
        ```python
        import cellengine
        exp = cellengine.get_experiment(name='my experiment')
        experiment.active_compensation = cellengine.UNCOMPENSATED
        ```
        """
```

Renders as:
![image](https://user-images.githubusercontent.com/469365/167750792-6559279d-6f24-4949-a6c3-4f7f385f9b4e.png)

Reply:
:+1: